File: 250424/007/007.java
import java.util.Scanner;

class Member {
    String code;
    String space;
    int time;

    public Member(String code, String space, int time) {
        this.code = code;
        this.space = space;
        this.time = time;
    }
}

public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        String code = sc.next();
        String space = sc.next();
        int time = sc.nextInt();

        Member member = new Member(code, space, time);
        System.out.println("secret code : " + member.code);
        System.out.println("meeting point : " + member.space);
        System.out.println("time : " + member.time);
        sc.close();
    }
}

File: 250424/007/007.cpp
#include <iostream>
#include <string>
using namespace std;

class Spy {
    public:
        string code;
        char place;
        int time;

        Spy(string code, char place, int time) {
            this->code = code;
            this->place = place;
            this->time = time;
        }
};

int main()
{
    ios::sync_with_stdio(false);
    cin.tie(0);
    cout.tie(0);

    string code;
    char place;
    int time;
    cin >> code >> place >> time;
    Spy spy = Spy(code, place, time);
    cout << "secret code : " << spy.code << "\n";
    cout << "meeting point : " << spy.place << "\n";
    cout << "time : " << spy.time;
    return 0;
}